README.md

```diff
@@ -35,59 +39,705 @@ var auth0 = require('auth0')({
 });
 ~~~
 
-### User's metadata
```

Comment:
I removed it since I haven't tested throughfully that logic, and I'm not sure it is fully working. Will bring it back when I've tested it, at least manually, to make sure the docs are up to date and the implementation working.